src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1206,6 +1206,10 @@ static bool addTriggeredSubscriptions_withCache
                                                            aList,
                                                            cSubP->subscriptionId,
                                                            cSubP->tenant);
+
+    // FIXME P10 #1316: expression needs to be stored in the cache
```

Comment:
P10 ...
Is this something you marked to not forget and then forgot?
Looks important enough to be included in this PR ...   ?